src/Worker/Notifier.php

```diff
@@ -388,10 +388,11 @@ function_exists('imap_open')
 
 				$relay_list_stmt = DBA::p(
 					"SELECT
-       					`batch`,
-       					ANY_VALUE(`id`) AS `id`,
-       					ANY_VALUE(`name`) AS `name`,
-       					ANY_VALUE(`network`) AS `network`
+						`batch`,
```

Comment:
Why the change of indentation? Readability as the indented part is the "select" part of the query?

Reply:
I just had replaced the spaces with tabs.

Reply:
Ah, there at the beginning of the line :) I had only checked the last indentation steps before the code.